training/config.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+
+# default config
+class BaseConfig(object):
+    DEBUG = False
+    # shortened for readability
+    #SECRET_KEY = '\xbf\xb0\x11\xb1\xcd\xf9\xba\x8bp\x0c...'
```

Comment:
Commented code. Maybe it's worth leaving the SECRET_KEY attribute and read its value from env?